### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dennisgl**
[Rendered](https://github.com/fusionjs/fusion-plugin-universal-events-react/blob/docs-fix-1/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,17 +83,17 @@ import UniversalEvents from 'fusion-plugin-universal-events';
 
 The plugin. Provides the [service API](#service-api). Typically should be registered to `UniversalEventsToken`.
 
-##### `UniversalEventsToken`
+#### Dependencies
+
+##### `FetchToken`
 
 ```js
-import {UniversalEventsToken} from 'fusion-plugin-universal-events';
```
Comment (KevinGrandon) [inline README.md]: Isn't this token still required?

Comment (KevinGrandon) [reply README.md]: Or not, because we just register this module as the token?

Comment (dennisgl) [reply README.md]: Yes, and it's already being addressed at the `Registration API` section

**Comment by dennisgl** [comment]
!merge

**Pull request merged**